gui/keybindings.go

```diff
@@ -564,6 +565,15 @@ func (g *Gui) commitContainerForm() {
 	g.pages.AddAndSwitchToPage("form", g.modal(form, 80, 11), true).ShowPage("main")
 }
 
+func (g *Gui) openBrowser() {
+	container := g.selectedContainer()
+	if container.WebPort == "" {
+		return
+	}
+	link := "http://" + container.WebPort + "/"
```

Comment:
I think the last `/` doesn't need

Reply:
well, it doesn't hurt but makes the URL complaint with URL spec
If you don't like it feel free to remove it